yorkie/metrics/metrics.go

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package metrics
+
+import "github.com/yorkie-team/yorkie/yorkie/metrics/prometheus"
+
+// metrics manages the metric information that Yorkie is trying to measure.
+type metrics struct {
+	Server    ServerMetrics
+	RPCServer RPCServerMetrics
+	DB        DBMetrics
+}
+
+func newMetrics() *metrics {
+	return &metrics{
+		Server:    prometheus.NewServerMetrics(),
+		RPCServer: prometheus.NewRPCServerMetrics(),
+		DB:        prometheus.NewDBMetrics(),
+	}
```

Comment:
Metrics are separated by modules(Server, RPCServer, DB). I think this is also a premature abstraction. First of all, how about combining the metrics into one instead of separating them?